### DISCUSSION TITLE
forge build --zkync doesnt work

### DISCUSSION THREAD

**Discussion by 48cork** [Cyfrin/foundry-full-course-cu#2592] [General]
sergio66@DESKTOP-L89DERR:/mnt/c/Users/Usuário/hello_foundry_v2$ forge build --zksync
error: unexpected argument '--zksync' found

  tip: to pass '--zksync' as a value, use '-- --zksync'

Usage: forge build [OPTIONS] [PATHS]...

For more information, try '--help'.
sergio66@DESKTOP-L89DERR:/mnt/c/Users/Usuário/hello_foundry_v2$ forge build -- --zksync
[⠑] Compiling...
No files changed, compilation skipped
sergio66@DESKTOP-L89DERR:/mnt/c/Users/Usuário/hello_foundry_v2$

**Comment by EngrPips** [comment]
Hello @48cork, run `foundryup-zksync`, then retry. Let's see how that plays out.
